### PR TITLE
add ability to ignore null properties

### PR DESCRIPTION
**Pull request by mgraf1**
Added a property `IgnoreNullProperties` that when true prevents null properties from being serialized.

### CODE REVIEW

File: src/RestSharp.SnakeCaseSerializer.Tests/SnakeCaseSerializationStrategyTests.cs [hunk 1]
```diff
@@ -7,11 +7,22 @@ namespace RestSharp.SnakeCaseSerializer.Tests
     [TestFixture]
     public class SnakeCaseSerializationStrategyTests
     {
-        public SnakeCaseSerializationStrategyTests()
+        private SnakeCaseSerializationStrategy strategyUnderTest;
```
Comment (mrstebo) [inline src/RestSharp.SnakeCaseSerializer.Tests/SnakeCaseSerializationStrategyTests.cs]: Naming format for private variables should be `_myVariable`. Although for some reason VS for Mac doesn't seem to like that format 😒

File: src/RestSharp.SnakeCaseSerializer.Tests/SnakeCaseSerializationStrategyTests.cs [hunk 2]
```diff
@@ -38,5 +49,42 @@ public void Should_HandleNumbers_In_PropertyName()
 
             Assert.AreEqual(expected, json);
         }
+
+        [Test]
+        public void WhenIgnoringNullValues_Should_Not_Serialize_Null_Properties()
```
Comment (mrstebo) [inline src/RestSharp.SnakeCaseSerializer.Tests/SnakeCaseSerializationStrategyTests.cs:54]: Great job! 👍

**Review by mrstebo — CHANGES_REQUESTED** [review]
Nicely implemented 👍 
Just need to change the naming format of that private variable and I'll approve 🎉

**Comment by mgraf1** [comment]
Thanks for the feedback! Change has been made.

**Comment by mrstebo** [comment]
Awesome. Just need to wait for AppVeyor to start working again so we know everything still works 😅

**Comment by mrstebo** [comment]
AppVeyor seems to not want to build this...but looks like it could be something on their end, so I'll just merge this in 👍

**Pull request merged**